dataactcore/scripts/initialize.py

```diff
@@ -88,6 +90,12 @@ def load_cfda(base_path):
     load_cfda_program(base_path)
 
 
+def load_object(base_path):
```

Comment:
The function name should have include object class, since`object` is programming concept and has a different meaning from `object_class`